examples/chip-tool/commands/common/Command.cpp

```diff
@@ -832,31 +817,36 @@ const char * Command::GetArgumentDescription(size_t index) const
     return nullptr;
 }
 
+const char * Command::GetReadOnlyGlobalCommandArgument() const
+{
+    if (GetAttribute())
+    {
+        return GetAttribute();
+    }
+
+    if (GetEvent())
```

Comment:
I don't understand this part.  Either GetAttribute returned non-null or this will return null...